Assets/MarkerMetro/Editor/MarkerMetroBuilder.cs

```diff
@@ -81,13 +81,43 @@ public static void DoBuildUniversal (string outputPath)
             {
                 throw new Exception(error); // ensures exit code != 0.
             }
+#if UNITY_EDITOR
```

Comment:
Classes inside the Editor folder are editor only, so no reason to have UNITY_EDITOR in the file.